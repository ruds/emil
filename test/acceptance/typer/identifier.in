it;
true;
it;
"hi there";
it;
it : string;
it : int;
