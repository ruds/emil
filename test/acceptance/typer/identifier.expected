@0001: ERROR
@0002:
val it : bool
@0003:
val it : bool
@0004:
val it : string
@0005:
val it : string
@0006:
val it : string
@0007: ERROR
@0008:
